app/src/main/java/eu/snigle/corsaire/proximite/CategoriesActivity.java

```diff
@@ -202,17 +202,24 @@ private void initCategories() {
                         add("atm");
                     }
                 });
-                put(getString(R.string.pharmacie), new ArrayList<String>() {
+                put(getString(R.string.sante), new ArrayList<String>() {
                     {
                         add("pharmacy");
-                        add("museum");
+                        add("hospital");
+                        add("dentist");
+                        add("doctor");
                     }
                 });
                 put(getString(R.string.restaurant), new ArrayList<String>() {
                     {
                         add("restaurant");
                     }
                 });
+                put(getString(R.string.hotel), new ArrayList<String>() {
+                    {
+                        add("lodging");
```

Comment:
All available types are here : https://developers.google.com/places/supported_types?hl=fr
There is establishment but there is too much results. With lodging I can have Gite and Hotels.